scripts/license_check.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+FILES=$(find . -type f -name "*.go")
+FAIL=0
+for file in  $FILES; do
+    if [ $(head -n1 "${file}"|grep package|wc -l) -eq 0 ]; then
```

Comment:
I would like to check for the specific license  that we use, as Helm does (also, the script is called license_check.sh but doesn't check licenses).

I would also prefer to check for either Rancher, SUSE or Rancher & SUSE copyright. As it is, leaving a newline on the beginning of the file will pass the lint.

Reply:
Makes sense, I will see if this can be invoked through github itself, this file is primarily intended to check for missing/wrong headers.